src/components/DocumentDashboard.vue

```diff
@@ -38,7 +38,16 @@
         </keep-alive>
       </ScrollingDocument>
       <DocumentLabelSets ref="labelSets" v-if="!editMode" />
-      <DocumentEdit ref="editView" v-else />
+      <!-- v-else prevents the error from being shown,
+      so it should be in the component, 
+      but then the image is not loaded -->
+      <DocumentEdit
+        ref="editView"
+        v-else
+        :showEditView="!!editMode"
```

Comment:
why is this necessary (showEditView)? can we just not use the editMode on the DocumentEdit component directly? because it's a property from the store so it's accessible (and it's already loaded there) :)